test/unit/lib/plugins/aws/package/compile/events/apiGateway/lib/usagePlan.test.js

```diff
@@ -363,6 +363,28 @@ describe('UsagePlan', () => {
     });
   });
 
+  it('Should have values for throttle and not quota', () => {
+    serverlessConfigurationExtension.provider.apiGateway = {
+      usagePlan: { throttle, quota: { limit: null, offset: null, period: null } },
```

Comment:
Instead of setting them all as `null`s, let's completely omit the `quota` property in configuration

Reply:
Hi @pgrzesik - sounds good. How about this?

```
serverlessConfigurationExtension.provider.apiGateway = {
      usagePlan: { throttle },
    };

...

expect(cfResources.ApiGatewayUsagePlan.Properties).to.not.contain(quota);
```

Reply:
Looks good - let's use: `to.not.have.property` assertion here

Reply:
Sounds good! I'll update it with that assertion.